views/pages/LoginPage.jsx

```diff
@@ -0,0 +1,281 @@
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Typography from '@material-ui/core/Typography'
+import TextField from '@material-ui/core/TextField'
+import Visibility from '@material-ui/icons/Visibility'
+import VisibilityOff from '@material-ui/icons/VisibilityOff'
+import IconButton from '@material-ui/core/IconButton'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import Button from '@material-ui/core/Button'
+import Snackbar from '@material-ui/core/Snackbar'
+import { Link } from 'react-router-dom'
+import { routes } from '../routes/routes'
+import api from '../api'
+
+const LoginPage = (props) => {
+  const [state, setState] = useState({
+    message: '',
+    username: '',
+    password: '',
+    showPassword: false,
+    open: false,
+    windowWidth: window.innerWidth,
+  })
+
+  const navigate = useNavigate()
+
+  const handleClick = () => {
+    setState((prevState) => {
+      return {
+        ...prevState,
+        open: true,
+      }
+    })
+  }
+
+  const handleCrumbs = () => {
+    setState((prevState) => {
+      return {
+        ...prevState,
+        open: false,
+        message: '',
+      }
+    })
+  }
+
+  const handleResize = () => {
+    setState({
+      windowWidth: window.innerWidth,
+    })
```

Comment:
do we need to keep the rest of state here?

```suggestion
    setState({
      ...state,
      windowWidth: window.innerWidth,
    })
```

Reply:
Thanks will remember this for the refactor of the login page. This is out of scope.

Reply:
I wonder if we just want to tackle this now because I believe it's a bug and it's an easy fix.

Reply:
I don't think this has to be updated now, the page will get refactored and this will probably go away